.lando.yml

```diff
@@ -3,13 +3,42 @@ recipe: drupal9
 config:
   webroot: web
   php: "8.1"
+  composer_version: '2.5.5'
```

Comment:
I believe this will work with just 2 instead of 2.5.5 and then it can float and we will continue to get composer bug fixes.